deploy/runtime/help/html/pv_system_design.html

```diff
@@ -691,30 +692,34 @@ <h4 class="p_Heading4"><span class="f_Heading4">Electrical Sizing Information an
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Total module area, m</span><span class="f_VariableName" style="font-size: 0.70rem; vertical-align: super;">2</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">This is an estimate of the area in the plane-of-array of all modules in the array. The total module area is not affected by the tilt angle or ground coverage ratio.</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">For the Detailed PV model, the total module area is the product of the area of a single module and the number of modules in the system. The area of a single module is defined on the <a href="pv_module.html" class="topiclink">Module</a> page.</span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">For PVWatts, the the total module area is the nameplate DC capacity of the array divided by the module efficiency. The module efficiency is determined by the module type as described above.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">For PVWatts, the the total module area is the nameplate DC capacity of the array divided by the module efficiency. The module efficiency is determined by the module type as described in the description of </span><span class="f_CHInterfaceElement">Module type</span><span class="f_VariableDescription">.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">AC capacity, MWac</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">The system AC capacity, or total inverter capacity in AC megawatts.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">For the Detailed PV model, the AC capacity is the product of the inverter's maximum AC power from the Inveter page and the number of inverters from the System Design page.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">For PVWatts, the AC capacity is the system nameplate capacity divided by the DC to AC ratio.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Land area per system capacity</span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWe</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWac</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity. Typical values range between 5 and 10 acres/MWac.</span></p>
```

Comment:
I might recommend knocking the lower limit down to 4. The latest report from the reV team has 4.3 acres/MWac (assuming an ILR of 1.34) https://www.nrel.gov/docs/fy24osti/87843.pdf